metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1585,6 +1642,27 @@ def _create_lineage_to_upstream_tables(
                 aspect=upstream_lineage,
             )
 
+    def _substitude_tableau_parameters(self, query: str) -> str:
```

Comment:
```suggestion
    def _clean_tableau_query_parameters(self, query: str) -> str:
```

this could also be a static/classmethod

Reply:
done https://github.com/datahub-project/datahub/pull/9838/commits/9be09d303e2e272a6a2ecae9427db4719f9b645d